src/App.tsx

```diff
@@ -101,40 +103,45 @@ const App = () => {
         <ThemeProvider theme={colorAdaptedTheme}>
           <SplashScreen>
             <PortalProvider>
-              <PortalHost name="browser-portal" />
-              <OnboardingProvider baseUrl={Config.ONBOARDING_API_URL}>
-                {connection && (
-                  <LockScreen>
-                    <AccountProvider
-                      extendConnection={false}
-                      commitment="confirmed"
-                      connection={connection}
-                    >
-                      <WalletConnectProvider>
-                        {accountsRestored && (
-                          <>
-                            <NavigationContainer
-                              theme={navTheme}
-                              linking={linking}
-                              ref={navigationRef}
-                            >
-                              <BalanceProvider>
-                                <NetworkAwareStatusBar />
-                                <RootNavigator />
-                              </BalanceProvider>
-                            </NavigationContainer>
-                            <SecurityScreen
-                              visible={
-                                appState !== 'active' && appState !== 'unknown'
-                              }
-                            />
-                          </>
-                        )}
-                      </WalletConnectProvider>
-                    </AccountProvider>
-                  </LockScreen>
-                )}
-              </OnboardingProvider>
+              <BottomSheetModalProvider>
```

Comment:
Is there a reason we didnt have this in the root before? I remember we moved it out a while back and put it into a bunch of individual files?

Reply:
Crap your right but I cant remember. Let me just move it back to the individual component